src/base/rss/private/rss_parser.cpp

```diff
@@ -206,11 +207,23 @@ namespace
     }
 }
 
-using namespace Rss::Private;
+using namespace RSS::Private;
+
+Parser::Parser(QString lastBuildDate)
+{
+    qRegisterMetaType<ParsingResult>();
```

Comment:
I just didn't want to do it in some external class. Admittedly, it's not the best solution. Best to fix it. Tell me please the best way. Maybe some static initializer?